src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        Task[] list = new Task[100];
+        int count = 0;
+
+        System.out.println("Hello! I'm Oriento.");
+        System.out.println("What can I do for you?");
+
+        while (true) {
+            String command = scanner.nextLine();
+            String[] parts = command.split(" ");
```

Comment:
Unspecific variable naming.
```suggestion
            String userCommand = scanner.nextLine();
            String[] userInputParts = userCommand.split(" ");
```